src/backend/distributed/sql/udfs/citus_conninfo_cache_invalidate/latest.sql

```diff
@@ -0,0 +1,9 @@
+ALTER FUNCTION master_conninfo_cache_invalidate()
```

Comment:
missed this last time: this should specify the pg_catalog schema name (mainly to prevent issues when someone, perhaps maliciously, created a master_conninfo_cache_invalidate in the public schema)